crates/bevy_openxr/examples/actions.rs

```diff
@@ -0,0 +1,164 @@
+// a simple example showing basic actions using the xr utils actions
+use bevy::{math::vec3, prelude::*};
+use bevy_openxr::{
+    add_xr_plugins,
+    helper_traits::ToQuat,
+    init::OxrTrackingRoot,
+    resources::OxrViews,
+};
+use bevy_xr_utils::xr_utils_actions::{
+    ActiveSet, XRUtilsAction, XRUtilsActionSet,
+    XRUtilsActionState, XRUtilsActionSystemSet, XRUtilsActionsPlugin, XRUtilsBinding,
+};
+
+fn main() {
+    App::new()
+        .add_plugins(add_xr_plugins(DefaultPlugins))
+        .add_plugins(bevy_xr_utils::hand_gizmos::HandGizmosPlugin)
+        .add_systems(Startup, setup_scene)
+        .add_systems(Startup, create_action_entities.before(XRUtilsActionSystemSet::CreateEvents))
+        .add_plugins(XRUtilsActionsPlugin)
+        .add_systems(
+            Update,
+            read_action_with_marker_component.after(XRUtilsActionSystemSet::SyncActionStates),
+        )
+        .add_systems(
+            Update,
+            handle_flight_input.after(XRUtilsActionSystemSet::SyncActionStates),
+        )
+        .run();
+}
+
+/// set up a simple 3D scene
+fn setup_scene(
+    mut commands: Commands,
+    mut meshes: ResMut<Assets<Mesh>>,
+    mut materials: ResMut<Assets<StandardMaterial>>,
+) {
+    // circular base
+    commands.spawn(PbrBundle {
+        mesh: meshes.add(Circle::new(4.0)),
+        material: materials.add(Color::WHITE),
+        transform: Transform::from_rotation(Quat::from_rotation_x(-std::f32::consts::FRAC_PI_2)),
+        ..default()
+    });
+    // cube
+    commands.spawn(PbrBundle {
+        mesh: meshes.add(Cuboid::new(1.0, 1.0, 1.0)),
+        material: materials.add(Color::rgb_u8(124, 144, 255)),
+        transform: Transform::from_xyz(0.0, 0.5, 0.0),
+        ..default()
+    });
+    // light
+    commands.spawn(PointLightBundle {
```

Comment:
No point lights, add an ambient light instead as a resource.

Reply:
I have changed to an ambient light for better performance like you asked.